src/containers/BookList.js

```diff
@@ -1,9 +1,11 @@
+/*eslint-disable*/
 import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import Book from '../components/Book';
 import CategoryFilter from '../components/CategoryFilter';
 import { removeBook, catFilter } from '../actions/index';
+// import CategoryFilter  from '../components/CategoryFilter';
```

Comment:
- [ ] [OPTIONAL]  Inline comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history